swagger/swagger.yaml

```diff
@@ -348,7 +348,7 @@ paths:
               schema:
                 $ref: "#/components/schemas/State"
         "400":
-          description: Invalid action or Session ID, or specified an intent action
+          description: Invalid action or Session ID, empty justification or specified an intent action
```

Comment:
Although this is true, we're not listing any other reasons the action can be invalid...so probably just leave it how it was before.